src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -228,7 +228,7 @@
 
 helps['cosmosdb sql database show'] = """
 type: command
-short-summary: Get the SQL database under an Azure Cosmos DB database account.
+short-summary: Show the details/properties of a SQL database under an Azure Cosmos DB database account.
```

Comment:
I mean you use one of {details, properties} in description according to your show scenario.